website/generate_sdk_markdown.py

```diff
@@ -2,14 +2,21 @@
 import importlib
 import inspect
 import os
-
 from types import FunctionType
-from typing import Dict, List, Literal, Optional
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    ForwardRef,
+    List,
+    Literal,
+    Optional,
+)
 
 from docstring_parser import parse
```

Comment:
As for now this is not a Terminal dependency.

Maybe we should add this as an extra so it will be easier to automate in a Github Action?